### ISSUE TITLE
[bert/pytorch] pytorch finetune on SQuAD

### ISSUE THREAD

**Issue by shunshen93** [NVIDIA/DeepLearningExamples#667]
Related to **[bert/pytorch]** 

I have loaded the checkpoint for bert-base-cased,  then finetine on SQuAD v1.1 as follow:

python   -m torch.distributed.launch --nproc_per_node=4 run_squad.py --init_checkpoint=bert-base-cased/pytorch_model.bin  --do_train  --train_file=download/squad/v1.1/train-v1.1.json  --train_batch_size=4 --do_predict --predict_file=download/squad/v1.1/dev-v1.1.json --predict_batch_size=4 --eval_script=download/squad/v1.1/evaluate-v1.1.py --do_eval  --bert_model=bert-base-cased  --learning_rate=3e-5  --seed=1  --num_train_epochs=2.0  --max_seq_length=384  --doc_stride=128  --output_dir=results/SQuAD  --vocab_file=download/google_pretrained_weights/cased_L-12_H-768_A-12/vocab.txt  --config_file=bert_config.json  --max_steps=-1 

but I got the F1 less then 80. could you please help to provide the hypeparameters?

**Comment by sharathts** [comment]
The base checkpoint uploaded on NGC is uncased. 

https://ngc.nvidia.com/catalog/models/nvidia:bert_pyt_ckpt_base_pretraining_amp_lamb/files

Use --do_lower_case and set --vocab_file=download/google_pretrained_weights/uncased_L-12_H-768_A-12/vocab.txt for right results. Other hyperparameters look good.
